src/super_gradients/recipes/arch_params/dekr_w32.yaml

```diff
@@ -0,0 +1,58 @@
+# This model config is mimicing the one from the original repo:
```

Comment:
Please rename this file to dekr_w32_arch_params.yaml to keep it consistent with the rest.